pyrbgt/environment.yml

```diff
@@ -3,9 +3,11 @@ channels:
   - conda-forge
   - pytorch3d
   - plotly
+  - nvidia
+  - pytorch
   - defaults
 dependencies:
-  - cudatoolkit=11.1
+  - nvidia::cudatoolkit
```

Comment:
Why remove the pinned cudatoolkit/pytorch version?

Reply:
Hmm, I was thinking if the dependency is largely just on some high level API, getting the latest version always could be better (if we detect a breaking change later with a newer version, then we can pin the specific version where the code is compatible). What do you think?

Reply:
Given the relative low amount of tests and users for this package and our difficulty reproducing previous results even given a Docker image, I'd err on the side of "reproducible when we pick it back up" by pinning specific versions (esp. since cuda/torch are very large packages). Once we increase the # of tests and users, we could consider this.